tools/rosbag/src/rosbag/bag.py

```diff
@@ -50,8 +50,14 @@
 import time
 import yaml
 
-from Crypto import Random
-from Crypto.Cipher import AES
+try:
+    from Cryptodome.Cipher import AES
+    from Cryptodome.Random import random
+
+except ImportError:
+    from Crypto.Cipher import AES
+    from Crypto.Random import random
```

Comment:
There is no need to support two different dependencies. Please remove the usage of `Crypto` in favor of the new dependency.